plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -91,6 +91,16 @@ const markedOptions = {
 				</${element}>
 			</div>
 			`;
+	},
+	table(header, body) {
+		return `
+			<div class="markdown table-wrapper">
+				<table class="table margin-top-s">
```

Comment:
Should we be adding `margin-top-s` here? Or should this work automatically?

Reply:
Without `margin-top` the rendered table would stick to the paragraph or whatever other element is rendered before, so some spacing between is needed.

Reply:
moved `margin-top-s` next to the `table-wrapper` so not it's on a correct element

Reply:
margin-top to the table-wrapper, so it's more clear?